CONTRIBUTING.md

```diff
@@ -23,7 +23,7 @@ The recommended way of setting your environment up is with [Anaconda](https://ww
 
 The project uses Python 3 (specified in the `python=3` bit of the commands below). [Download Anaconda](https://www.continuum.io/downloads) and create an _environment_ for the project.
 
-The version of Anaconda doesn't have to be 3 as `conda` environments manage Python version as well. But if you're going to a fresh install we recommend Anaconda 3 — it's newer and it makes no sense to have Python 2 as your default Anaconda Python.
```

Comment:
> You can use either Anaconda 2 and Anaconda 3, since conda environments manage Python versions. But if you're going to do a fresh install, we recommend Anaconda 3 — it's newer and it makes no sense to have Python 2 as your default Anaconda Python.

👍 Go for it!